### ISSUE TITLE
AH64DBLKII: Wrong value for PLT_EUFD_LINE3 when preset mode active

### ISSUE THREAD

**Issue by emcek** [DCS-Skunkworks/dcs-bios#506]
This is what I got in cockpit:
![image](https://github.com/DCS-Skunkworks/dcs-bios/assets/475312/27857b1c-e795-466c-9142-858b36acd322)
This what DCS-BIOS report:
```
DEBUG   | AH64DBLKII PLT_EUFD_LINE1  : "LOW ROTOR RPM     |RECTIFIER 2 FAIL  |PRESET TUNE VHF   "
DEBUG   | AH64DBLKII PLT_EUFD_LINE2  : "ENGINE 2 OUT      |GENERATOR 2 FAIL  |!PRESET 1 127.500 "
DEBUG   | AH64DBLKII PLT_EUFD_LINE3  : "ENGINE 1 OUT      |AFT FUEL LOW      $$$$$$$$$$$$$$.000 "
DEBUG   | AH64DBLKII PLT_EUFD_LINE4  : "                  |FORWARD FUEL LOW  | PRESET 3 136.000 "
DEBUG   | AH64DBLKII PLT_EUFD_LINE5  : "                  |                  | PRESET 4 127.000 "
DEBUG   | AH64DBLKII PLT_EUFD_LINE6  : "                                     | PRESET 5 125.000 "
DEBUG   | AH64DBLKII PLT_EUFD_LINE7  : "                                     | PRESET 6 121.000 "
DEBUG   | AH64DBLKII PLT_EUFD_LINE8  : "~<>VHF*  127.500   MAN               | PRESET 7 141.000 "
DEBUG   | AH64DBLKII PLT_EUFD_LINE9  : " ==UHF*  240.000   PRE 2         L2  | PRESET 8 128.000 "
DEBUG   | AH64DBLKII PLT_EUFD_LINE10 : " ==FM1*   30.015   PRE 3    NORM L3  | PRESET 9 130.000 "
DEBUG   | AH64DBLKII PLT_EUFD_LINE11 : " ==FM2*   30.020   PRE 4         L4  | PRESET10 130.000 "
DEBUG   | AH64DBLKII PLT_EUFD_LINE12 : " ==HF *    2.0000A          LOW       |                 "
DEBUG   | AH64DBLKII PLT_EUFD_LINE13 : "                                                        "
DEBUG   | AH64DBLKII PLT_EUFD_LINE14 : "FUEL    0          XPNDR S 1200 A NORM        04:03:20 Z"
```

BTW: Minor additional issue `PLT_EUFD_LINE6` and `PLT_EUFD_LINE7` do not have pipe (`|`) character and `PLT_EUFD_LINE12` has extra one.

DCS-BIOS: 0.7.50 (eb0ed557)
DCS openbeta: 2.9.2.49940

**Comment by charliefoxtwo** [comment]
Well this is interesting.

When I initially implemented this, there wasn't a way to tell if the display was in preset mode. Perhaps there is now? I'll look into it. Thank you for the report!

**Comment by emcek** [comment]
I know few mount ago it was working ok. Do not remember when it was, there were few Apache update since then.

**Comment by charliefoxtwo** [comment]
I was able to reproduce this. As I suspected, ED changed some of the exports. Overall the new ones make a bit more sense so it's probably for the best. Wasn't too difficult to fix - should be merged in the next day or two.

Here's what it looks like now (at least in my tool - I still haven't changed that it exports $$$ for those bar lines)

Main
![image](https://github.com/DCS-Skunkworks/dcs-bios/assets/30303272/77bca148-0d31-4db1-88fb-7a34be279ead)

Preset
![image](https://github.com/DCS-Skunkworks/dcs-bios/assets/30303272/cf3655d0-84fd-49d3-9233-2a63596d69fd)
